packages/scroller/src/vaadin-scroller.d.ts

```diff
@@ -15,7 +16,7 @@ import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mix
  * </vaadin-scroller>
  * ```
  */
-declare class Scroller extends ThemableMixin(ElementMixin(HTMLElement)) {
+declare class Scroller extends TabindexMixin(ThemableMixin(ElementMixin(HTMLElement))) {
```

Comment:
Small difference here: In the `.d.ts` file we added `TabindexMixin`, in the `.js` file we added `FocusMixin`.

Reply:
Done